core-methods/src/main/java/com/mercadopago/sdk/android/coremethods/ui/components/textfield/pcitextfield/PCIFieldState.kt

```diff
@@ -0,0 +1,41 @@
+package com.mercadopago.sdk.android.coremethods.ui.components.textfield.pcitextfield
+
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.Stable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.saveable.Saver
+import androidx.compose.runtime.saveable.rememberSaveable
+import androidx.compose.runtime.setValue
+
+/**
+ * This class holds the input data of secure fields. It makes sure the fields are PCI Compliant.
```

Comment:
What do you think? Since the field itself is responsive for only part of the PCI good standards. 

```suggestion
 * This class holds the input data of secure fields, safeguarding it against exposure and helping maintain a PCI Compliant environment.
```

Reply:
It makes sense. updated